src/nvim/popupmenu.c

```diff
@@ -976,6 +1154,9 @@ void pum_set_event_info(dict_T *dict)
   (void)tv_dict_add_float(dict, S_LEN("row"), r);
   (void)tv_dict_add_float(dict, S_LEN("col"), c);
   (void)tv_dict_add_nr(dict, S_LEN("size"), pum_size);
+  if (pum_selected >= 0) {
+    (void)tv_dict_add_nr(dict, S_LEN("selected"), pum_selected);
+  }
```

Comment:
I don't think this work when `ext_popupmenu` is enabled, while `complete_info().popup` does work, so I don't think this should be added.

Reply:
correct. I didn't notice complete_info at the time. Found it later. So this is a duplicate